docs/man-pages/condor_token_request.rst

```diff
@@ -0,0 +1,136 @@
+
+
+*condor_token_request*
+======================
+
+request a token from a remote daemon
+:index:`condor_token_request<single: condor_token_request; HTCondor commands>`\ :index:`condor_token_request command`
+
+Synopsis
+--------
+
+**condor_token_request** [**-identity** *user@domain*] [**-authz** *authz* ...]
+[**-lifetime** *value*]
+[**-pool** *pool_name*] [**-name** hostname] [**-type** *type*]
+[**-token** *filename*]
+
+**condor_token_request** [**-help** ]
+
+Description
+-----------
+
+*condor_token_request* will request an authentication token from a remote
+daemon.  Unlike *condor_token_fetch*, the user doesn't need to have an existing
+authenticated identity with the remote daemon when using *condor_token_request*.
+Token requests must be approved by the daemon's administrator using
+*condor_token_request_approve*.
+
+If the request is successfully enqueued, the request ID will be printed to ``stderr``;
+the administrator will need to know the ID to approve the request.
+
+The token request mechanism provides a powerful way to bootstrap authentication
+in a HTCondor pool - a remote user can request an identity, call the system
+administrator on the telephone to confirm the authenticity of the request, and
```

Comment:
How about "verify the authenticity of the request out-of-band"? Doesn't need to be phone.

Reply:
Young people these days, not wanting to call infrastructure on the phone....